utils/exportPdfs.ts

```diff
@@ -114,10 +114,20 @@ const generateDictionaryPDFs = async (country: CountryVariant) => {
   }
 };
 
+const generateTalesPDFs = async (country: CountryVariant) => {
+  const stories: string[] = ['pernikova-chaloupka', 'dvanact-mesicku', 'cervena-karkulka', 'kolobok', 'husy-lebedi', 'ivasik-telesik'];
```

Comment:
Is it necessary to have  this hard-coded? same keys has STORIES object in storyStore.ts